File: app/src/main/java/dev/ragnarok/fenrir/mvp/view/search/IPhotoSearchView.java
package dev.ragnarok.fenrir.mvp.view.search;

import java.util.ArrayList;

import dev.ragnarok.fenrir.model.Photo;

public interface IPhotoSearchView extends IBaseSearchView<Photo> {
    void displayGallery(int accountId, ArrayList<Photo> photos, int position);
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/fragment/search/photosearch/IPhotoSearchView.kt
package dev.ragnarok.fenrir.fragment.search.photosearch

import dev.ragnarok.fenrir.fragment.search.abssearch.IBaseSearchView
import dev.ragnarok.fenrir.model.Photo

interface IPhotoSearchView : IBaseSearchView<Photo> {
    fun displayGallery(accountId: Long, photos: ArrayList<Photo>, position: Int)
}